app/Http/Controllers/ActivityController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Activity;
+use Illuminate\Http\Request;
+use App\User;
+use Auth;
+use File;
+use Image;
+use Validator;
+use Gate;
+
+class ActivityController extends Controller
+{
+        /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // get user
+        $user = Auth::user();
+        if (Gate::allows('admin-only', auth()->user())) {
+            $activities = Activity::orderBy('id', 'desc')->paginate(10);
+            //return $activities;
+            return view('activities.index', compact('activities'));
+        }
+     return back()->with('error', __('messages.not_allowed'));
+
+    }
+
+    public function deleteAll()
+    {
+        // get user
+        $user = Auth::user();
```

Comment:
Initialisation of variables should be separate by a white line with the next lines of code